### DISCUSSION TITLE
Storage down - 502

### DISCUSSION THREAD

**Discussion by tjelz** [vercel/community#4640] [Help] [answered]
### Summary

My database is down and cannot be accessed via Storage.
I get a 502 error page (BAD_GATEWAY)

An unexpected error has occurred.

This is breaking my website as it relies on that data.
Is this a global issue, or just related to me?

### Steps to Reproduce

Try to press Storage on the vercel website.

**Comment by sergiustan77** [comment]
Doesn't work for me either, 504 error code
`NeonDbError: Server error (HTTP status 504): {"code":504,"message":"HTTP-Connection timed out, execution time exeeded 15 seconds"}`

**Comment by sergiustan77** [comment]
Seems like it's back 😄

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. I've investigated and can confirm that this was a service-side transient issue. Our upstream provider, Neon, is aware. It is back to normal now, but please do let me know if you continue to have trouble with it. 🙏
